tests/api.treatments.test.js

```diff
@@ -39,7 +41,11 @@ describe('Treatment API', function ( ) {
       self.ctx.data.treatments[1].insulin.should.equal(2);
       self.ctx.data.treatments[2].carbs.should.equal(30);
 
-      done();
+      //if travis is slow the 2 posts take long enough that 2 data-loaded events are emitted
+      if (!doneCalled) done();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'done'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1505937736/issues/source?bid=2452379&fileBranchId=3005271#l45)